### ISSUE TITLE
[Request] Random spoofing only per-domain not per-request

### ISSUE THREAD

**Issue by tegru** [dillbyrne/random-agent-spoofer#215]
Not an expert but I would think that constantly changing x-forwarder-for, screensize etc. would look quite suspicious in logs and certainly bring attention to the user.

It would be better to keep a random value as long as browsing the same domain. I can see this being much more difficult to implement though...

**Comment by kekkc** [comment]
This would be fantastic. Instead of changing every 10min blindly, RAS would store the values for the single host and use a new value for a new host. A different host can then use different values, but I have no idea if storing the values in some DB / JSON is possible. A new session / browser restart should then also trigger new values for the stored hosts.

**Comment by kekkc** [comment]
Think the problem is that some settings are global (javascript navigator properties, changes in about:config...). Either we have an inproper spoofing that is easily discovered and just changes the user agent, which can be domain specific, or we have a correct spoofing with the correct JS values. I doubt that it's possible to use this domain specific, but let's see if dillbyrne has an info.

**Comment by dillbyrne** [comment]
@kekkc You are correct some of the profile properties are global.

There is no improper spoofing, RAS spoofs values in three places, 

1. In the built in browser preferences where the values exist
2. In the headers so they match the profile
3. In the injected script to add additional values that can't be spoofed using the built in options.

Domain based spoofing can only be done on the header and injection script level which is how we apply the whitelist. The built in prefs are the guaranteed to work, in the event the script is somehow bypassed. 

 Once the multiprocess functionality lands in stable and I know where I stand with the API in respect to linking requests to tabs I may be able to enhance the current functionality but for now it .

I will update #214 When I know more. Closing this as it is a duplicate of that issue
